### PR TITLE
Steps removed from Upgrading Content Hosts

### PR DESCRIPTION
**Pull request by bangelic**
The first seven steps of the Procedure
have been removed. The reason for this
change is due to the change in version
independent Satellite Client
repository. As a result, these steps
are no longer needed.

* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.4/Katello 4.6
* [x] Foreman 3.3/Katello 4.5
* [x] Foreman 3.2/Katello 4.4
* [ ] Foreman 3.1/Katello 4.3
* For Foreman 3.0 or older, please create a separate PR.
* We do not accept PRs for Foreman 2.3 or older.

### CODE REVIEW

**Comment by bangelic** [comment]
@evgeni Can you confirm that is good so we can merge it?

File: guides/doc-Upgrading_and_Updating/topics/upgrading_content_hosts.adoc [hunk 1]
```diff
@@ -28,14 +28,7 @@ If these files have been deleted, they must be restored from a backup in order f
...
 .Procedure
-. In the {ProjectWebUI}, navigate to *Hosts* > *Content Hosts* and select the Content Hosts that you want to upgrade.
-. From the *Select Action* list, select *Manage Repository Sets*.
-. From the *Repository Sets Management* list, select the *{ProjectName} Tools {ProjectVersionPrevious}* checkbox.
-. From the *Select Action* list, select *Override to Disabled*, and click *Done*.
-. When the process completes, on the same set of hosts from the previous steps, from the *Select Action* list, select *Manage Repository Sets*.
-. From the *Repository Sets Management* list, select the *Red Hat {project-client-name}* checkbox.
-. From the *Select Action* list, select *Override to Enabled*, and click *Done*.
-. When the process completes, on the same set of hosts from the previous steps, from the *Select Action* list, select *Manage Packages*.
+. From the *Select Action* list, select *Manage Packages*.
```
Comment (adamlazik1) [inline guides/doc-Upgrading_and_Updating/topics/upgrading_content_hosts.adoc:31]: Isn't the base location missing? What I mean is the `navigate to ... > ...` got removed too.

**Comment by evgeni** [comment]
As noted in https://github.com/theforeman/foreman-documentation/pull/1719 this *whole section* is irrelevant for Satellite 6.12+ as we do not require users to upgrade the client tooling at the same time as they upgrade Satellite.

6.11 is when we introduced this new style, so 6.12 is the first upgrade where they do not have to perform that.

**Comment by bangelic** [comment]
> As noted in #1719 this _whole section_ is irrelevant for Satellite 6.12+ as we do not require users to upgrade the client tooling at the same time as they upgrade Satellite.
> 
> 6.11 is when we introduced this new style, so 6.12 is the first upgrade where they do not have to perform that.

@evgeni Should I change the branch to 3.3?

File: guides/doc-Upgrading_and_Updating/topics/upgrading_content_hosts.adoc (hunk 1 above)
Comment (bangelic) [reply guides/doc-Upgrading_and_Updating/topics/upgrading_content_hosts.adoc]: @adamlazik1 Per Evgeni's direction, I have removed the entire section.

**Comment by bangelic** [comment]
@evgeni Is this ready to be merged?

**Comment by evgeni** [comment]
> > > As noted in #1719 this _whole section_ is irrelevant for Satellite 6.12+ as we do not require users to upgrade the client tooling at the same time as they upgrade Satellite.
> > > 6.11 is when we introduced this new style, so 6.12 is the first upgrade where they do not have to perform that.
> > 
> > 
> > @evgeni Should I change the branch to 3.3?
> 
> I don't think so. From my understanding, this should be merged to master and then cherry-picked to 3.{5,4,3}. However; if the procedure is no longer relevant, than maybe the CLI procedure isn't too? Or perhaps @evgeni meant the entire module including introduction?

Correct, I meant that the whole chapter/module "3.4. Upgrading Content Hosts" is irrelevant for Satellite deployments and can be removed (or rather it's inclusion conditionalized on `ifndef::satellite[]` and all `ifdef::satellite` blocks inside it removed)

File: guides/doc-Upgrading_and_Updating/master.adoc [hunk 2]
```diff
@@ -59,7 +59,7 @@ endif::[]
 // Upgrading Capsule Server
 include::topics/upgrading_capsule_server.adoc[leveloffset=+2]
 
-ifdef::katello,orcharhino,satellite[]
+ifndef::satellite[]
```
Comment (evgeni) [inline guides/doc-Upgrading_and_Updating/master.adoc:62]:
```suggestion
ifdef::katello,orcharhino[]
```

Sorry, I didn't see it was already conditioned (and you did it the right way around in the other places)

File: guides/doc-Upgrading_and_Updating/topics/upgrading_content_hosts.adoc [hunk 3]
```diff
@@ -27,30 +27,9 @@ Failure to retain these files during an upgrade causes the upgrade to fail.
...
-+
-. Until https://bugzilla.redhat.com/show_bug.cgi?id=1649764[BZ#1649764] is resolved, from the *Update* list, you must select *via remote execution*.
-This is required because if you update the package using the Katello agent, the package update disrupts the communication between the client and {Project} or {SmartProxyServer}, which causes the update to fail.
-For more information, see {ManagingHostsDocURL}Configuring_and_Setting_Up_Remote_Jobs_managing-hosts[Configuring and Setting Up Remote Jobs] in _{ManagingHostsDocTitle}_.
-endif::[]
-
 [[cli-upgrading_content_hosts]]
 .CLI Procedure
 . Log into the client system.
-ifdef::satellite[]
```
Comment (evgeni) [inline guides/doc-Upgrading_and_Updating/topics/upgrading_content_hosts.adoc:53]: This means the following now gets rendered for Katello/Orcharino, but it doesn't apply there. Remove all the content between the `ifdef` and `endif` please

**Review by evgeni — APPROVED** [review]

**Comment by evgeni** [comment]
To github.com:theforeman/foreman-documentation.git
   0f0875532..a554c7ba2  3.3 -> 3.3
   1f7d3b2c0..8a79ab8af  3.4 -> 3.4
   d20f4e4d5..eec7ba0f2  3.5 -> 3.5


It had a conflict when picking to 3.2, and it's also not really relevant there as it's a Satellite only change and there is no Satellite based on 3.2

**Pull request merged**